### PR TITLE
Update PS version in changelog

### PR DESCRIPTION
**Pull request by matks**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Update PS version in changelog
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26830)

### CODE REVIEW

**Review by sowbiba — DISMISSED** [review]

File: docs/CHANGELOG.txt [hunk 1]
```diff
@@ -21,7 +21,7 @@ needs please refer to https://devdocs.prestashop.com/ for more information.
 @copyright Since 2007 PrestaShop SA and Contributors
 @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
 
-Release Notes for PrestaShop 1.7
+Release Notes for PrestaShop 8
```
Comment (PierreRambaud) [inline docs/CHANGELOG.txt]:
```suggestion
Release Notes for PrestaShop
```

**Review by PierreRambaud — CHANGES_REQUESTED** [review]
Two solutions imho:
- removing all 1.7 data in this file
- Ignore PrestaShop version

**Comment by matks** [comment]
> * Ignore PrestaShop version

Simple and safe: let's do that

**Review by sowbiba — APPROVED** [review]

**Review by PierreRambaud — APPROVED** [review]

**Comment by PierreRambaud** [comment]
Thank you @matks

**Pull request merged**